extras/pre-00.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I think we dropped the extras/ directory, so I'd say we can drop this file

Reply:
Dropped